src/middlewares/logger.ts

```diff
@@ -2,6 +2,7 @@ import datadogLogger from "../services/datadog";
 import { Env } from "../types/Env";
 import { Context, Next } from "hono";
 import { Var } from "../types/Var";
+import { waitUntil } from "src/utils/wait-until";
```

Comment:
typescript isn't happy with this absolute paths. I'd make everything relative. :heart: relative imports, hate magic :smile: 

I think this is enough

```suggestion
import { waitUntil } from "../utils/wait-until";
```